doc/rados/operations/add-or-rm-mons.rst

```diff
@@ -81,67 +77,65 @@ and root permissions.
 Adding a Monitor (Manual)
 -------------------------
 
-This procedure creates a ``ceph-mon`` data directory, retrieves the monitor map
-and monitor keyring, and adds a ``ceph-mon`` daemon to your cluster.  If
-this results in only two monitor daemons, you may add more monitors by
-repeating this procedure until you have a sufficient number of ``ceph-mon`` 
-daemons to achieve a quorum.
+The procedure in this section creates a ``ceph-mon`` data directory, retrieves
+both the monitor map and the monitor keyring, and adds a ``ceph-mon`` daemon to
+the cluster. The procedure might result in a Ceph cluster that contains only
+two monitor daemons. To add more monitors until there are enough ``ceph-mon``
+daemons to establish quorum, repeat the procedure.
```

Comment:
This implies that two mons is never enough to form quorum, which contradicts above descriptions of mon count.  When adding a mon, won't there *always* be quorum unless one or more mons already in the map is down?

Reply:
I'm going to deal with this in a future, quorum-focused PR. We'll get to the bottom of this, just not in the line-edit PR.